src/fides/api/util/consent_util.py

```diff
@@ -54,9 +58,15 @@ def filter_privacy_preferences_for_propagation(
     return filtered_on_use
 
 
-def should_opt_in_to_service(
```

Comment:
renamed this existing fn to better reflect what it returns, which is a Tuple of:
1. Some kind of consent preference(s), for opt-in/out this is just True or False, for notice-based this is a mapping
2. The complete list of preferences that apply for sake of reporting